### PR TITLE
Without plugin

### PR DESCRIPTION
**Pull request by parimal786**

### CODE REVIEW

**Comment by apremalal** [comment]
Parima,

here you have deleted wp-config.php, what you have to do is not to commit that file by just ignoring while commiting.

File: wp-content/themes/hotelmaster-v2-01/stylesheet/style-custom.css [hunk 1]
```diff
@@ -1,18 +1,15 @@
 body{ background: url('http://themes.goodlayers2.com/hotelmaster/wp-content/uploads/2015/03/bg-pattern.png'); }
 html.ltie9 body, body{ min-width: 1140px; } .container{ max-width: 1140px; } .gdlr-caption-wrapper .gdlr-caption-inner{ max-width: 1140px; }
-/*.body-wrapper.gdlr-boxed-style{
-	!*max-width: 1220px;*!
-} */
-.body-wrapper.gdlr-boxed-style .gdlr-header-wrapper{ max-width: 1220px; margin: 0px auto; }
+.body-wrapper.gdlr-boxed-style{ max-width: 1220px; } .body-wrapper.gdlr-boxed-style .gdlr-header-wrapper{ max-width: 1220px; margin: 0px auto; }
```
Comment (apremalal) [inline wp-content/themes/hotelmaster-v2-01/stylesheet/style-custom.css]: Why did you remove this?.  I have commented this 'cause with this the websites responsiveness get lost in the mobile device.

Comment (apremalal) [inline wp-content/themes/hotelmaster-v2-01/stylesheet/style-custom.css]: Basically the content has to fill the whole window. Please fix it.

**Comment by parimal786** [comment]
I did not even touch any core files, It was a simple text change and some image issues which were fixed via dashboard, Also it is always a good practice to create and use the child theme so that css of main theme is not effected. I will go through what and why things changed in style-custom.css(if we change any style in wp it is usually done only in style.css whereas this is core theme file where we never try to access).

**Comment by apremalal** [comment]
Hi Parmial,
Please check how it got changed. As it clearly says it's coming from your commit.
